2025/03/delhi-fish-market-consent-reel/writeup.md

```diff
@@ -0,0 +1,27 @@
+# Writeup
+
+So whatever happened to this girl and the camera man in this video is wrong in every way.
+
+But I heard this girl saying in the video "Sir app chhu nahin sakte" intending that they cannot touch them and come closer to them without their consent. 
+
+But the same creator who is upset and concerned when her rights are violated will later go and consume animal product for which animals are violated day in and day out 
+
+They are tide hands are being put into their rectum hands in the vagina to make them pregnant they show every sign as if they are also saying"sir aap chu nahi sakte" but there voices are unheard the signs ignored. 
```

Comment:
Please change this to:
The animal's legs are tied, and someone shoves their hands into that animal's rectum, and a rod in their vagina to make...
```suggestion
The animal's legs are tied, and someone shoves their hands into that animal's rectum, and a rod in their vagina to make them pregnant.

They show every sign as if they are also saying "sir aap chu nahi sakte", but there voices are unheard the signs ignored.
```